src/android/CHIPTool/app/src/main/res/layout/cluster_interaction_history_item_info.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+
+  <TextView
+      android:id="@+id/historyClusterNameTv"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:maxLines="2"
+      android:ellipsize="end"
+      android:layout_marginStart="8dp"
+      android:layout_marginTop="8dp"
+      android:textAlignment="viewStart"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toStartOf="@+id/historyCommandNameTv"
+      app:layout_constraintWidth_percent="0.4" />
```

Comment:
Shouldn't these percents add up to 1.0? If you want a gap between the textViews, margin is a better way of doing it.